src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -19,87 +21,95 @@
         }
 
-        Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
+        Property propertyToEdit = lastShownList.get(index.getZeroBased());
+        Property editedProperty = createdEditedProperty(propertyToEdit, editPropertyDescriptor);
 
-        if (!personToEdit.isSamePerson(editedPerson) && model.hasPerson(editedPerson)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
+        if (!propertyToEdit.isSameProperty(editedProperty) && model.hasProperty(editedProperty)) {
+            throw new CommandException(MESSAGE_DUPLICATE_PROPERTY);
         }
 
-        model.setPerson(personToEdit, editedPerson);
-        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson));
+        model.setProperty(propertyToEdit, editedProperty);
+        model.updateFiltedPropertyList(PREDICATE_SHOW_ALL_PROPERTIES);
+        return new CommandResult(String.format(MESSAGE_EDIT_PROPERTY_SUCCESS, editedProperty));
     }
 
     /**
-     * Creates and returns a {@code Person} with the details of {@code personToEdit}
+     * Creates and returns a {@code Property} with the details of {@code propertyToEdit}
      * edited with {@code editPersonDescriptor}.
      */
-    private static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
-        assert personToEdit != null;
-
-        Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
-        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
-        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
-        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
-        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
-
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+    private static Property createdEditedProperty(Property propertyToEdit, EditPropertyDescriptor
+            editPersonDescriptor) {
+        assert propertyToEdit != null;
+
+        Name updatedName = editPersonDescriptor.getName().orElse(propertyToEdit.getName());
+        Seller updatedSeller = editPersonDescriptor.getSeller().orElse(propertyToEdit.getSeller());
+        Price updatedPrice = editPersonDescriptor.getPrice().orElse(propertyToEdit.getPrice());
+        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(propertyToEdit.getPhone());
+        Email updatedEmail = editPersonDescriptor.getEmail().orElse(propertyToEdit.getEmail());
+        Address updatedAddress = editPersonDescriptor.getAddress().orElse(propertyToEdit.getAddress());
+        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(propertyToEdit.getTags());
```

Comment:
Can you remove remaining occurrences of person here and where mentioned by Guangxuan (excluding test code)?
Use the IDE to help you